CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Copyright: (C) 2016 iCub Facility - Fondazione Istituto Italiano di Tecnologia
+# Authors: Francesco Romano <francesco.romano@iit.it>
+# CopyPolicy: Released under the terms of the GNU LGPL v2.0+
+
+cmake_minimum_required(VERSION 3.0)
+project(xsens-mvn)
+
+# Add local CMake files
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
+else()
+    set(CMAKE_CXX_STANDARD 11)
+endif()
+
+# Find YARP and include the CMake code to compile plugins
+find_package(YARP REQUIRED)
+list(APPEND CMAKE_MODULE_PATH ${YARP_MODULE_PATH})
+
+include(YarpPlugin)
+include(YarpInstallationHelpers)
+set(YARP_FORCE_DYNAMIC_PLUGINS TRUE)
+# I want to install plugins in PREFIX/yarp.
+#yarp_configure_external_installation(xsens-mvn-driver WITH_PLUGINS)
+
+# Compile the plugins by default
+set(COMPILE_BY_DEFAULT ON)
+
+set(CMAKE_POSITION_INDIPENDENT_CODE ON)

```

Comment:
ahah the nice thing is that I looked for the documentation with the correct term.. but I wrote the wrong one in the cmake 🤣